spec/features/admin_items_view_spec.rb

```diff
@@ -26,7 +26,7 @@
     click_link_or_button("Create Item")
     fill_in('item[title]', with: "Racoon Ragu" )
     fill_in('item[description]', with: "A bandit of a meal" )
-    fill_in('item[price]', with: 5.34)
+    fill_in('item[price_in_dollars]', with: 5.34)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.